packages/orbit-components/src/Card/__tests__/index.test.tsx

```diff
@@ -66,7 +60,7 @@ describe("Card", () => {
 
     render(
       <Card>
-        <CardSection title="kek" onClick={onClick} />
+        <CardSection title="kek" expandable onClick={onClick} />
```

Comment:
`onClick` shouldn't work when `CardSection` is non-expandable. components shouldn't have extraneous event handlers for best practice reasons, and because it breaks RSCs

Reply:
I believe a Card can be clickable for some action (eg: route to another page).
Our tracking tool shows no usages of `onClick` on `CardSection`, though.

Reply:
ah okay I misunderstood the purpose of `onClick` then. alright I will return `onClick` as it was before, though normally this would be solved by composition to also allow hrefs, for example:

```
<Card>
  <Link href={`/product/${id}`}>
    <CardSection title={title}>
      <Product id={id} />
    </CardSection>
  </Link>
</Card>
```